src/components/FlexItem.js

```diff
@@ -77,6 +99,14 @@ FlexItem.propTypes = {
    * Specify vertical gap between items
    */
   gapVertical: PropTypes.oneOf(Object.keys(GAP)),
+  /**
+   * Specify if height or width 100% should be applied to the div
+   */
+  stretched: PropTypes.bool,
+  /**
+   * Specify the min width for this flex item. Defaults to not setting this value
```

Comment:
nit: Not necessary to document default, since its already generated part of the defaultProps in the storybook